net.sourceforge.Chessx.json

```diff
@@ -31,6 +31,10 @@
                     "sha256": "ad951910a3bd53f5655f9e85101e7969b536078473e09a2a7dded6ef171a68ed",
                     "strip-components": 2
                 },
+                {
+                    "type": "patch",
```

Comment:
There is [already](https://sourceforge.net/projects/chessx/files/chessx/1.5.0/) a version `1.5.0`...so **maybe** this patch is not needed.